tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNet/ApiController_ExecuteAsync_Integration.cs

```diff
@@ -92,7 +92,9 @@ public class ApiController_ExecuteAsync_Integration
             var controllerContext = (IHttpControllerContext)state.State;
 
             // some fields aren't set till after execution, so populate anything missing
-            AspNetWebApi2Integration.UpdateSpan(controllerContext, scope.Span, (AspNetTags)scope.Span.Tags, Enumerable.Empty<KeyValuePair<string, string>>());
+            var span = scope.Span as IHasTags;
+            var aspNetTags = span.Tags as AspNetTags;
```

Comment:
Not related to this particular cast but I wonder if we shouldn't have a cast helper that would log if null to gain visibility.